### ISSUE TITLE
Codec original

### ISSUE THREAD

**Issue by iAldrich23xX** [ImNotYour5G/SynapsePM#1]
The codec original is https://github.com/iAldrich23xX/SynapsePM.git please not copy

**Comment by ImNotYour5G** [comment]
Your code is by NukkitX and NukkitX has the code by iTXTech!

**Comment by iAldrich23xX** [comment]
yes but use mi page host.firebt.net in /utils/DataPacketEidReplacer.php please quit

**Comment by iAldrich23xX** [comment]
or the delete and not upload updates :)
